de.metas.report/report-service/src/main/java/de/metas/adempiere/report/jasper/ADJasperFiller.java

```diff
@@ -58,31 +58,61 @@ private ADJasperFiller()
 	public JasperPrint fillReport(
 			final JasperReport jasperReport,
 			final Map<String, Object> parameters,
-			final Connection connection,
+			final Connection source,
```

Comment:
pls avoid having 2 fillReport methods implementation.
If u need another method with less params, just implement it by calling the other `fillReport` method.